chain/actors/builtin/README.md

```diff
@@ -0,0 +1,29 @@
+# Actors
```

Comment:
This file was copied over from lotus: https://github.com/filecoin-project/lotus/blob/master/chain/actors/builtin/README.md

Everything else in this PR was created via code gen.